fifeforspark/utils.py

```diff
@@ -105,29 +105,6 @@ def create_example_data2(
         StructField('feature_5', StringType(), True)])
     return sc.createDataFrame(values, schema = schema)
 
-def import_data_file(file_path: str = "Input_Data") -> pyspark.sql.DataFrame:
```

Comment:
I think reading to pandas first may not work. Instead let's look at the Spark.read.csv("") option (and similar methods)